pkg/collector/python/loader.go

```diff
@@ -308,39 +308,8 @@ func reportPy3Warnings(checkName string, checkFilePath string) {
 	status := a7TagUnknown
 	metricValue := 0.0
 	if checkFilePath != "" {
-		// __file__ return the .pyc file path
-		if strings.HasSuffix(checkFilePath, ".pyc") {
-			checkFilePath = checkFilePath[:len(checkFilePath)-1]
-		}
-
-		if strings.TrimSpace(pkgconfigsetup.Datadog().GetString("python_version")) == "3" {
-			// the linter used by validatePython3 doesn't work when run from python3
-			status = a7TagPython3
-			metricValue = 1.0
-		} else {
-			// validatePython3 is CPU and memory hungry, make sure we only run one instance of it
-			// at once to avoid CPU and mem usage spikes
-			linterLock.Lock()
-			warnings, err := validatePython3(checkFilePath)
```

Comment:
@pgimalac Drive-by conceptual question: Should we have this as part of Diagnose tooling?

Reply:
It's an interesting suggestion, but the linter helps transitioning from python2 to python3 so I'm not quite sure it's useful for users of Agent 7 (this code never runs in agent 7 btw), it would require adding an extra python dependency (`pylint`) to the package, and lastly it has been removed from Agent 7 because [it could not run on Python 3](https://github.com/pylint-dev/pylint/issues/1969#issuecomment-377127442) (seems to have been fixed in more recent versions though, but I haven't tested)

Reply:
I suppose my curiosity is if we should include a regular linter (`pylint`) warnings/errors (not `py3k` specifically) in the diagnose command output to find issues with problematic Python code either in our integrations or custom ones. Fair point that the extra dep is definitely a barrier in terms of footprint constraints though.